### PR TITLE
chore: regenerate scaffolded files

### LINKED ISSUE #255: Custom document conflict resolution mechanism
refs https://github.com/eHealthAfrica/direct-delivery-dashboard/issues/119

### PR DESCRIPTION
**Pull request by tlvince**
Updates scaffolded files from generator-gulp-angular to v1.0.

Most visible changes:

* No more auto-modification of files under `src`, see https://github.com/Swiip/generator-gulp-angular/pull/217, closes #255

* `src/components` now lives under `src/app/components`, see https://github.com/Swiip/generator-gulp-angular/issues/559

* `gulp test-watch` -> `gulp test:auto`

Refs: https://github.com/eHealthAfrica/direct-delivery-dashboard/pull/150

### CODE REVIEW

File: src/app/components/delivery/cancel-delivery/cancel-delivery.service.spec.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ describe('cancelDeliveryService', function() {
     it('Should return TRUE if status equals DELIVERY_STATUS.CANCELED_OTHER', function(){
       expect(cancelDeliveryService.validateCancelReport(facRnd)).toBeFalsy();
       facRnd.status = deliveryStatus.CANCELED_OTHER;
+      facRnd.cancelReport.note = 'other';
```
Comment (tlvince) [inline src/app/components/delivery/cancel-delivery/cancel-delivery.service.spec.js]: @jofomah validateCancelReport expects there to be a note, so I presume this is the correct behaviour? (not sure why it passed previously)

File: src/app/components/delivery/deliver-items/deliver-items.controller.spec.js [hunk 2]
```diff
@@ -10,12 +10,14 @@ describe('DeliverItemsCtrl', function () {
 
   beforeEach(inject(function ($controller, _$state_, _dailyDeliveryMock_, _deliveryService_, _log_, FACILITY_ID, _$rootScope_) {
 
+    var dailyDeliveryMock = angular.copy(_dailyDeliveryMock_);
```
Comment (tlvince) [inline src/app/components/delivery/deliver-items/deliver-items.controller.spec.js]: This was being mutated between runs

File: src/app/components/delivery/delivery.service.spec.js [hunk 3]
```diff
@@ -125,7 +125,7 @@ describe('deliveryService', function () {
       var packedProduct = angular.copy(deliveryMock.facilityRounds[0].packedProduct[0]);
       packedProduct.onHandQty = 20;
       //expected(70) - onhand(20) = 50 when round off by presentation(20) = 60
-      var expectedDeliveredQty = 60;
+      var expectedDeliveredQty = 50;
```
Comment (tlvince) [inline src/app/components/delivery/delivery.service.spec.js]: @jofomah wasn't sure about this one.

**Comment by tlvince** [comment]
Updated daily delivery mocks where status from pending to upcoming 1st.

**Comment by tlvince** [comment]
Local builds (`gulp build`) and Cordova (`gulp cordova`) pass, but I have not tested an APK on a device.

**Comment by tlvince** [comment]
@musamusa could you take a look at this? The main motivation is to fix the issue of `index.html` always being auto-modified.

**Comment by musamusa** [comment]
@tlvince ok

**Comment by tlvince** [comment]
:star2:

**Pull request merged**